internal/delete_handler.go

```diff
@@ -448,10 +453,40 @@ func (h *DeleteHandler) findDependantBackups(target BackupObject) []BackupObject
 	var curr BackupObject
 	for len(queue) > 0 {
 		curr, queue = queue[0], queue[1:]
-		relatedBackups = append(relatedBackups, curr)
+		dependantBackups = append(dependantBackups, curr)
 		queue = append(queue, incrementsByBackup[curr.GetBackupName()]...)
 	}
-	return relatedBackups
+	return dependantBackups
+}
+
+func DeleteObjectsWhere(
```

Comment:
Moved this function here from `storage/folder.go` to make it possible to log storage names inside. It wasn't possible to reference `multistorage` package from `storage` one because of a cyclic dependency.